models/Tree.php

```diff
@@ -251,7 +251,7 @@ public function activateNode($currNode = true)
                 }
             }
         }
-        if (!empty($this->nodeActivationErrors)) {
+        if ($this->nodeActivationErrors) {
```

Comment:
`nodeActivationErrors` and `nodeRemovalErrors` is an array even if it is initialized ... and checking if it is not `empty` is ok.